### PR TITLE
Add throttleCPU command

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

This is a command that we will add to Sauce Labs in order to allow people running extended debugging in Chrome to throttle the CPU.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

This is currently behind a beta flag therefor we can hold off merging the PR until it is released.

### CODE REVIEW

File: packages/webdriver/protocol/saucelabs.json [hunk 1]
```diff
@@ -108,6 +108,29 @@
       }]
     }
   },
+  "/session/:sessionId/sauce/ondemand/throttle/cpu": {
```
Comment (mgrybyk) [inline packages/webdriver/protocol/saucelabs.json]: @christian-bromann does this returns something?

**Comment by mgrybyk** [comment]
CI is failing because of 
```
    Expected: > 299
    Received:   299
      218 |     const start = Date.now()
      219 |     await traceGatherer.onLoadEventFired()
    > 220 |     expect(Date.now() - start).toBeGreaterThan(299)
```

File: packages/webdriver/protocol/saucelabs.json [hunk 2]
```diff
@@ -108,6 +108,29 @@
...
+    "POST": {
+      "command": "throttleCPU",
+      "description": "You can throttle the CPU in DevTools to understand how your page performs under that constraint.",
+      "ref": "https://wiki.saucelabs.com/display/DOCS/Custom+Sauce+Labs+WebDriver+Extensions+for+Network+and+Log+Commands#CustomSauceLabsWebDriverExtensionsforNetworkandLogCommands-ThrottleNetworkCapabilities",
```
Comment (mgrybyk) [inline packages/webdriver/protocol/saucelabs.json]: the links leads to network throttling instead of cpu

**Comment by mgrybyk** [comment]
@christian-bromann  https://wiki.saucelabs.com/display/DOCS/Sauce+Performance+CLI+%28Speedo%29+-+Configuration+Options#SaucePerformanceCLI(Speedo)-ConfigurationOptions-CPUThrottlingConfigurations

**Comment by christian-bromann** [comment]
> @christian-bromann [wiki.saucelabs.com/display/DOCS/Sauce+Performance+CLI+%28Speedo%29+-+Configuration+Options#SaucePerformanceCLI(Speedo)-ConfigurationOptions-CPUThrottlingConfigurations](https://wiki.saucelabs.com/display/DOCS/Sauce+Performance+CLI+%28Speedo%29+-+Configuration+Options#SaucePerformanceCLI(Speedo)-ConfigurationOptions-CPUThrottlingConfigurations)

Yup, the new driver was released last Thursday. We can merge this PR.

File: packages/webdriver/protocol/saucelabs.json (hunk 1 above)
Comment (christian-bromann) [reply packages/webdriver/protocol/saucelabs.json]: nope

**Comment by christian-bromann** [comment]
@webdriverio/project-committers any objections merging this? Please approve if not. Cheers!

File: packages/webdriver/protocol/saucelabs.json (hunk 2 above)
Comment (mgrybyk) [reply packages/webdriver/protocol/saucelabs.json]: The link is still invalid

Comment (christian-bromann) [reply packages/webdriver/protocol/saucelabs.json]: right, sorry, fixing it!

Comment (christian-bromann) [reply packages/webdriver/protocol/saucelabs.json]: updated link

**Pull request merged**